test/integration/common/serverspec/spec_helper.rb

```diff
@@ -0,0 +1,3 @@
+require 'serverspec'
```

Comment:
Yeah yeah. I'll updated it to inspec at a later date.  It will happen just not right now.